concepts/05 UI Components/SelectBox/00 Getting Started with SelectBox/20 Handle the Value Change Event.md

```diff
@@ -1,82 +1,5 @@
-Implement the [onValueChanged](/api-reference/10%20UI%20Components/dxSelectBox/1%20Configuration/onValueChanged.md '/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#onValueChanged') function to perform an action when a user selects an item. In the code below, this function logs the selected item's `ID` and `Name`.
+#include tutorials-editors-handle-value-change with {
+    output: "the IDs of the currently and previously selected items"
```

Comment:
Does it mean that your component stores the two selection states - the current and previous ones?

Reply:
I would not say about selection state here. A selection state is a state of a component, a thing that shows whether it is selected. But here we talk about selected values. In my opinion, these are two different things.

Reply:
Alright, I'm not going to suggest any changes here. I'm just trying to figure out of what do we speak here. What are previously selected items, for example?